core/api/Dockerfile-websocket

```diff
@@ -30,4 +30,4 @@ USER 1000
 ARG COMMITHASH
 ENV COMMITHASH ${COMMITHASH}
 
-CMD ["lib/servers/ws-server.js"]
+CMD ["dist/servers/ws-server.js"]
```

Comment:
Double-checking, is this dockerfile used? This has been a bug that made the file unusable for the last few weeks